src/types/types.ts

```diff
@@ -0,0 +1,25 @@
+export interface Todo {
+  id: number;
+  userId: number;
+  title: string;
+  completed: boolean;
+}
+
+export type SendedTodo = Omit<Todo, 'id'>;
+
+export type LoadTodos = () => void;
+
+export enum FILTERS {
+  all = 'All',
+  completed = 'Completed',
+  active = 'Active',
+}
+
+export enum ErrorTypes {
+  none = '',
+  get = 'unable to get todos',
```

Comment:
It would be better if the messages started with an uppercase letter. Change it everywhere, please 
```suggestion
  get = 'Unable to get todos',
```